R/iSEE-main.R

```diff
@@ -946,7 +946,7 @@ iSEE <- function(
         }
     }
 
-    # Linked selection structure observers for the heatmaps.
+    # Brush structure observers for the heatmaps.
```

Comment:
This is really a zoom observer, nothing to do with brushing here.

Reply:
Indeed, now that you pointed it out, I noticed the .zoomClick under the .brushField.
Fixed. Thanks!